modules/github-repository/repository.tf

```diff
@@ -1,4 +1,5 @@
 resource "github_repository" "repository" {
+  count                  = var.create_repository == null ? 1 : 0
```

Comment:
why var.create_repository == null ? 1 : 0 
and not just var.create_repository ? 1 : 0
?